openBCIBoard.js

```diff
@@ -1149,8 +1148,8 @@ function OpenBCIFactory() {
             while (readingPosition <= bytesToRead - k.OBCIPacketSize) {
                 if (data[readingPosition] === k.OBCIByteStart) {
                     var rawPacket = data.slice(readingPosition, readingPosition + k.OBCIPacketSize);
-                    this.emit('rawDataPacket',rawPacket);
                     if (data[readingPosition + k.OBCIPacketSize - 1] === k.OBCIByteStop) {
+                        this.emit('rawDataPacket',rawPacket);
```

Comment:
so now it just makes sure the stop byte is there before emitting the packet? how does this take care of the bug (packet dropped every 30ms)?